scene/3d/physics/collision_object_3d.cpp

```diff
@@ -30,6 +30,7 @@
 
 #include "collision_object_3d.h"
 
+#include "collision_shape_3d.h"
```

Comment:
This is leftover from some prototyping and is not needed as the Physics Server is consulted for access to the AABB.